pennylane/operation.py

```diff
@@ -120,6 +121,20 @@
 from .variable import Variable
 
 
+#=============================================================================
+# Wire types
+#=============================================================================
+
+Wires = IntEnum("Wires", {"Any": -1, "All": 0})
```

Comment:
Is this idiomatic? I would actually prefer to not define classes in a functional way if possible:
```suggestion
class Wires(IntEnum):
    Any = -1
    All = 0
```
Or is there some advantage in calling `IntEnum` to build the Wires object?

Reply:
No advantage to choosing the functional approach over the class based approach. Just came down to personal preference :)